modules/@apostrophecms/seo-fields-doc-type/index.js

```diff
@@ -42,6 +44,42 @@ module.exports = {
           }
         }
       };
+
+      if (self.options.seoCanonicalTypes
+        && Array.isArray(self.options.seoCanonicalTypes)
+        && self.options.seoCanonicalTypes.length) {
+
+        const req = options.apos.task.getReq();
+
+        self.options.seoCanonicalTypes.forEach(canonicalType => {
+          const name = canonicalType.startsWith('@apostrophecms/') ? canonicalType.split('@apostrophecms/')[1] : canonicalType;
+          const fieldName = `_seoCanonical${_.capitalize(_.camelCase(name))}`;
+          const moduleName = options.apos.modules[canonicalType] && options.apos.modules[canonicalType].label
+            ? options.apos.modules[canonicalType].label
+            : name;
+          const label = req.t('aposSeo:canonicalModule', { type: _.startCase(req.t(moduleName)) });
+          const help = req.t('aposSeo:canonicalModuleHelp', { type: _.lowerCase(self.__meta.name) });
```

Comment:
do you really need lodash here?

Reply:
I use 3 times, and it's useful. What do you suggest instead? Have my own helpers in this module?